nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EncryptContent.java

```diff
@@ -68,70 +73,78 @@
-        .name("public-keyring-file")
-        .displayName("Public Keyring File")
-        .description("In a PGP encrypt mode, this keyring contains the public key of the recipient")
-        .required(false)
-        .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-        .build();
+            .name("public-keyring-file")
+            .displayName("Public Keyring File")
+            .description("In a PGP encrypt mode, this keyring contains the public key of the recipient")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
     public static final PropertyDescriptor PUBLIC_KEY_USERID = new PropertyDescriptor.Builder()
-        .name("public-key-user-id")
-        .displayName("Public Key User Id")
-        .description("In a PGP encrypt mode, this user id of the recipient")
-        .required(false)
-        .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-        .build();
+            .name("public-key-user-id")
+            .displayName("Public Key User Id")
+            .description("In a PGP encrypt mode, this user id of the recipient")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
     public static final PropertyDescriptor PRIVATE_KEYRING = new PropertyDescriptor.Builder()
-        .name("private-keyring-file")
-        .displayName("Private Keyring File")
-        .description("In a PGP decrypt mode, this keyring contains the private key of the recipient")
-        .required(false)
-        .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-        .build();
+            .name("private-keyring-file")
+            .displayName("Private Keyring File")
+            .description("In a PGP decrypt mode, this keyring contains the private key of the recipient")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
     public static final PropertyDescriptor PRIVATE_KEYRING_PASSPHRASE = new PropertyDescriptor.Builder()
-        .name("private-keyring-passphrase")
-        .displayName("Private Keyring Passphrase")
-        .description("In a PGP decrypt mode, this is the private keyring passphrase")
-        .required(false)
-        .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-        .sensitive(true)
-        .build();
-
+            .name("private-keyring-passphrase")
+            .displayName("Private Keyring Passphrase")
+            .description("In a PGP decrypt mode, this is the private keyring passphrase")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .sensitive(true)
+            .build();
+    public static final PropertyDescriptor RAW_KEY_HEX = new PropertyDescriptor.Builder()
+            .name("raw-key-hex")
+            .displayName("Raw key (hexadecimal)")
```

Comment:
I copied the casing and style for `name` from `private-keyring-passphrase` which was an existing property descriptor. Or were you talking about the `displayName` attribute? I will fix that.